src/BuildScriptGenerator/DotNetCore/DotnetCorePlatform.cs

```diff
@@ -40,24 +42,26 @@ internal class DotNetCorePlatform : IProgrammingPlatform
         /// <param name="environmentSettingsProvider">The environment settings provider.</param>
         /// <param name="logger">The logger of .NET platform.</param>
         /// <param name="detector">The detector of .NET platform.</param>
-        /// <param name="buildOptions">The build options for BuildScriptGenerator.</param>
+        /// <param name="commonOptions">The build options for BuildScriptGenerator.</param>
```

Comment:
This one may be 'cliOptions'?

Reply:
sure. changed in rest of the places too.